INSTALL.rst

```diff
@@ -87,65 +89,80 @@ For example, on Debian GNU/Linux, you can install them as follows:
                         rabbitmq-server \
                         redis-server
 
-and do a system install for the Sass preprocessor by following `Sass web guide <http://sass-lang.com/install>`_ and running:
+Now, add the following lines in your "elasticsearch.yml" (for
+Debian GNU/Linux the full path is
+``/etc/elasticsearch/elasticsearch.yml``):
 
 .. code-block:: shell
 
-  sudo npm install -g sass node-sass clean-css uglify-js requirejs
-
+  # MY CONFIGS
 
-Installation
-^^^^^^^^^^^^
+  cluster.name: cap
+  discovery.zen.ping.multicast.enabled: false
+  http.port: 9200
+  http.publish_port: 9200
 
-Let's start by creating a new virtual environment that will hold our CAP
-v3.0 instance:
+Finally, do a system install (see below for how to do a local install
+instead) for the Sass preprocessor by following
+`Sass web guide <http://sass-lang.com/install>`_ and running:
 
 .. code-block:: shell
 
-   mkvirtualenv cap
+  sudo npm install -g node-sass clean-css uglify-js requirejs
 
-Start redis server in the background:
+Installation
+^^^^^^^^^^^^
+
+Let's start by cloning the repository:
```

Comment:
It's not about branching it's just about independency of code and libraries used.
If you clone the repository you have the code there. Then, if you want to run CAP locally you can create a virtualenv with all the necessary libraries contained inside and workon that within the repository. It's essentially the same as before only that you decide from within your repository which virtualenv/libraries you want to use any time. If you want to use a new version of a library you can create a new virtualenv with that library version and still run the old commits with the old virtualenv. So if it fails with the new library (or you're still working on it but need to run code with the older version) you can always switch back to using the old one without any effort.
It comes with far less chance to mess up your own setup.

We can also test with different python versions if we have one virtualenv per version we want to support and then we just switch between these with workon and deactivate.